### DISCUSSION TITLE
A problem with 3.3.3

### DISCUSSION THREAD

**Discussion by Guba** [pawelsalawa/sqlitestudio#4224] [General]
Hello to the Community.
I want to expose a problem that I'm having. When I start the program (Ver. 3.3.3) two messages appear:
[17:04:49] Cannot load plugin DbSqlite2. Error details: Cannot load library C:\Program Files\SQLiteStudio\plugins\DbSqlite2.dll: Unknown error 0x000000c1.
[17:04:49] Cannot load plugin DbSqliteSystemData. Error details: Cannot load library C:\Program Files\SQLiteStudio\plugins\DbSqliteSystemData.dll: Unknown error 0x000000c1.
What am I doing worng ?

**Comment by pawelsalawa** [comment]
`DbSQLite2` is an old plugin, from previous version of SQLiteStudio and it will not work with new version.
The same goes for the other one - `DbSqliteSystemData`. It's old plugin and will not work with new SQLiteStudio.

For the `DbSqliteSystemData` you can use the `WxSQLite3` plugin (which you most likely already have loaded), as it supports the database format of `DbSqliteSystemData`.

The SQLite2 (handled so far by `DbSQLite2` plugin) is no longer supported. It was barely used and was needed a lot of maintenance effort, thus it's discontinued. If you really need SQLite2 support, you need to go with older SQLiteStudio version.

Errors are harmless, but if you would like to get rid of these error messages, go to the configuration dialog, go to "Plugins" page, find these 2 plugins there and uncheck them.

**Reply by telmob** [reply]
Hi! I've tried to make sense of this and reading all the posts regarding this, and could not find a solution.
I have a database that is encrypted with System.Data.SQLite and i could open it in the old version (3.2.1), but i can't now with the WxSQLite3 plugin, even though the option is there. I would gladly use the older version, but there's a DPI problem with it, and it looks tiny in my monitor.
Can you guys check it out please?

**Reply by pawelsalawa** [reply]
Try experimenting with these pragmas: https://utelle.github.io/SQLite3MultipleCiphers/docs/ciphers/cipher_aes256cbc/
They should be added to the Database Dialog, where you pick the plugin, password, etc. For example:
![image](https://user-images.githubusercontent.com/481846/150487495-a870f589-a5bb-4db5-944e-3779434189fd.png)

**Reply by telmob** [reply]
Fantastic! Thank you so much!
You guys are great 💯

**Comment by Guba** [comment]
Thank you for your support!

Sergio


Il giorno mar 5 ott 2021 alle ore 12:38 Paweł Salawa <
***@***.***> ha scritto:
